docker-compose.yaml

```
@@ -9,40 +9,32 @@
-      - electronyat-shop
+        - electronyat-shop
+        - shared
     labels:
       - "com.centurylinklabs.watchtower.enable=true"
 
 volumes:
-    db-data:
-        driver: local
-        name: db-data
-    images-data:
+    images:
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 4

(indentation)